### ISSUE TITLE
Crash while taking photo using camera. 

### ISSUE THREAD

**Issue by prsnt** [nguyenhoanglam/ImagePicker#24]
while cheking in kitkat. i found an issue when i clicked photo using camera it shows double photos with same capture. and another issue is my application is crashed while taking picture in camera(in LOLLIPOP).

**Comment by danndi** [comment]
Hello,
photos, that are created from this library are shown twice in my device too. I am using Android 5.0.1. In the regular gallery application the photo is shown only once. 

Also the exception is thrown. This is the message:

`ava.lang.NullPointerException: Attempt to invoke virtual method 'android.os.Message android.os.Handler.obtainMessage()' on a null object reference
                                                                      at com.nguyenhoanglam.imagepicker.activity.ImagePickerActivity$ImageLoaderRunnable.run(ImagePickerActivity.java:545)
                                                                      at java.lang.Thread.run(Thread.java:818)`

**Comment by nguyenhoanglam** [comment]
@danndi Did you check version 1.1.1 yet? This version fixed _NullPointerException_ . I'll try to fix the duplicate bugs soon!

**Comment by danndi** [comment]
Thank you for your response. I have tried the new version and no crash occurred yet :). Good job and good luck with the duplicate bug.

**Comment by nguyenhoanglam** [comment]
The new version has a new feature: _**folder mode**_. Look more detail in README file.

**Comment by danndi** [comment]
I am trying it now. It works great. Thank you for this library. 

Anyway I have tried to make a photo using the intent from your app. The photo is duplicated in the ImagePicker gallery. In the gallery application of the device it is only once. So I deleted this photo in the gallery. Then I tried your ImagePicker again and the photo is there but only once (although it should be removed completely). Maybe this will help you to find the issue.
